_help/troubleshooting.md

```diff
@@ -325,3 +325,29 @@ Check your `ulimit -a`. Many systems have a 1024 open file descriptor limit by d
 ```
 
 Make sure to raise your ulimit. Example: `ulimit -n 16384`
+
+## Headless TCP Services Losing Connection from Istiofied Containers
+
+Envoy is reset every 15 minutes by the `istio-ca` deployment, as a part
```

Comment:
If `istio-ca` is deployed, Envoy is restarted every 15 minutes to refresh the certificates, causing the disconnection of TCP streams or long-running connections.

Reply:
@andraxylia Do you want me to make these changes?

Reply:
yes please do

Reply:
Yes, please rephrase, sorry if that was not clear. Envoy is not restarted by the istio-ca, but by the pilot agent.